front/src/applications/operationalStudies/__tests__/sampleData.ts

```diff
@@ -344,3 +345,194 @@ export const electrificationRangesLarge: ElectrificationRangeV2[] = [
+  id: 70,
+  timetable_id: 5,
+  train_name: 'tooFast',
+  labels: [],
+  rolling_stock_name: '1TGV2N2',
+  start_time: '2024-08-02T12:00:00Z',
+  path: [
+    {
+      id: 'id209',
+      deleted: false,
+      uic: 87575001,
+      secondary_code: 'BV',
+    },
+    {
+      id: 'id249',
+      deleted: false,
+      uic: 87485276,
+      secondary_code: '00',
+    },
+    {
+      id: 'id210',
+      deleted: false,
+      uic: 87485003,
+      secondary_code: 'BV',
+    },
+  ],
+  schedule: [
+    {
+      at: 'id249',
+      arrival: 'PT2580S',
+      stop_for: 'P0D',
+      on_stop_signal: false,
+      locked: false,
+    },
+    {
+      at: 'id210',
+      arrival: null,
+      stop_for: 'P0D',
+      on_stop_signal: false,
+      locked: false,
+    },
+  ],
+  margins: {
+    boundaries: ['id210'],
+    values: ['20%', '0%'],
+  },
+  initial_speed: 0,
+  comfort: 'STANDARD',
+  constraint_distribution: 'MARECO',
+  speed_limit_tag: null,
+  power_restrictions: [],
+  options: {
+    use_electrical_profiles: true,
+  },
+};
+
+export const trainSummaryTooFast: Extract<SimulationSummaryResult, { status: 'success' }> = {
+  status: 'success',
+  length: 145084000,
+  time: 4400445,
+  energy_consumption: 3253493611.0058613,
+  path_item_times_final: [0, 2579714, 4400445],
+  path_item_times_provisional: [0, 3079443, 4900174],
+  path_item_times_base: [0, 2567248, 4084949],
+};
+
+export const trainScheduleNotHonored: TrainScheduleResult = {
+  id: 72,
+  timetable_id: 5,
+  train_name: 'notHonored',
+  labels: [],
+  rolling_stock_name: '1TGV2N2',
+  start_time: '2024-08-02T12:00:00Z',
+  path: [
+    {
+      id: 'id209',
+      deleted: false,
+      uic: 87575001,
+      secondary_code: 'BV',
+    },
+    {
+      id: 'id249',
+      deleted: false,
+      uic: 87485276,
+      secondary_code: '00',
+    },
+    {
+      id: 'id210',
+      deleted: false,
+      uic: 87485003,
+      secondary_code: 'BV',
+    },
+  ],
+  schedule: [
+    {
+      at: 'id249',
+      arrival: 'PT1800S',
+      stop_for: 'P0D',
+      on_stop_signal: false,
+      locked: false,
+    },
+    {
+      at: 'id210',
+      arrival: null,
+      stop_for: 'P0D',
+      on_stop_signal: false,
+      locked: false,
+    },
+  ],
+  margins: {
+    boundaries: [],
+    values: ['0%'],
+  },
+  initial_speed: 0,
+  comfort: 'STANDARD',
+  constraint_distribution: 'MARECO',
+  speed_limit_tag: null,
+  power_restrictions: [],
+  options: {
+    use_electrical_profiles: true,
+  },
+};
+
+export const trainSummaryNotHonored: Extract<SimulationSummaryResult, { status: 'success' }> = {
+  status: 'success',
+  length: 145084000,
+  time: 4084949,
+  energy_consumption: 3729835446.4172735,
+  path_item_times_final: [0, 2567248, 4084949],
+  path_item_times_provisional: [0, 2567248, 4084949],
+  path_item_times_base: [0, 2567248, 4084949],
+};
+
+export const trainScheduleHonoredAndNotTooFast: TrainScheduleResult = {
```

Comment:
Maybe trainScheduleHonored may be enough for the name :)
```suggestion
export const trainScheduleHonored: TrainScheduleResult = {
```

Reply:
Done